pkg/aws/client.go

```diff
@@ -1262,20 +1267,25 @@ func (r CustomRetryer) ShouldRetry(req *request.Request) bool {
 	}
 	logger := logging.NewLogger()
 	if strings.Contains(req.Error.Error(), "Throttling") {
-		logger.Warn(fmt.Sprintf("Throttling Rate limit exceeded. Retrying Attempt: '%v' of '%v'",
-			req.RetryCount+1, r.MaxRetries()))
+		logger.Warn(fmt.Sprintf(
+			"Throttling Rate limit exceeded. Retrying [Request ID: %s / %s %s]: %v/%v",
+			req.RequestID,
+			req.HTTPRequest.Method,
+			req.HTTPRequest.URL.Host,
+			req.RetryCount+1,
+			r.MaxRetries()))
 	}
 
 	return r.DefaultRetryer.ShouldRetry(req)
 }
 
-func buildCustomRetryer() CustomRetryer {
+func BuildCustomRetryer() CustomRetryer {
```

Comment:
what's the purpose to change the buildCustomRetryer() func to a public one, it will be used somewhere else?

Reply:
Hi Marco,
I had to do that in order to make the function reusable within the test file.